src/gluonts/distribution/binned.py

```diff
@@ -204,6 +204,9 @@ def hybrid_forward(
     ) -> Tuple[Tensor, Tensor]:
         ps = self.proj(x)
         reshaped_probs = ps.reshape(shape=(-2, -1, self.num_bins), reverse=1)
+        bin_centers = F.broadcast_add(
+            bin_centers.expand_dims(axis=0), ps.zeros_like()
```

Comment:
Why not doing
```python
bin_centers = mx.nd.broadcast_like(bin_centers.expand_dims(axis=0), ps)
```

Reply:
I simplified it to

```
F.broadcast_add(bin_centers, ps.zeros_like()
```

Unfortunately, `broadcast_like` doesn't work here, because the two tensors have to have the same number of axes for that. The `broadcast_add` works as long as the last dimensions match.